docs/modules/ROOT/pages/user-guide/generate-java-from-wsdl.adoc

```diff
@@ -59,5 +59,5 @@ quarkus.cxf.codegen.wsdl2java.bar-params.additional-params = -wsdlLocation,wsdl/
 ----
 
 TIP: Add `io.quarkiverse.cxf:quarkus-cxf-xjc-plugins` dependency to your project to be able to use
```

Comment:
Can you add link to https://quarkiverse.github.io/quarkiverse-docs/quarkus-cxf/1.5/reference/extensions/quarkus-cxf-xjc-plugins.html somewhere in the end of this TIP ?